source/PhysicalQuantity/Power.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace PhpUnitsOfMeasure\PhysicalQuantity;
+
+use PhpUnitsOfMeasure\AbstractPhysicalQuantity;
+use PhpUnitsOfMeasure\UnitOfMeasure;
+use PhpUnitsOfMeasure\HasSIUnitsTrait;
+
+class Power extends AbstractPhysicalQuantity
+{
+    use HasSIUnitsTrait;
+
+    protected static $unitDefinitions;
+
+    protected static function initialize()
+    {
+        // watt
+        $kiloWatt = UnitOfMeasure::nativeUnitFactory('W');
+        $kiloWatt->addAlias('watt');
+        $kiloWatt->addAlias('watts');
+        static::addUnit($kiloWatt);
+
+        static::addMissingSIPrefixedUnits(
+            $kiloWatt,
+            1,
+            '%pW',
+            [
+                '%Pwatt',
+                '%Pwatts',
+            ]
+        );
+
+        $metricHorsePower = UnitOfMeasure::linearUnitFactory('PS', 735.49875);
```

Comment:
Would it be a kind of configuration unit declaration that would allow easier changes and less code?

If you need help to implement/think this, let me know.